src/components/Dropdown/Dropdown.tsx

```diff
@@ -74,6 +74,12 @@ const StyledMenuButton = styled(Menu.Button)`
   cursor: pointer;
   padding-left: 0.75rem;
   overflow: hidden;
+  &:hover {
+    background: ${({ theme }) => theme.colors.secondaryVariant};
+    transition-property: background;
+    transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
+    transition-duration: 150ms;
```

Comment:
I like that you always looking for these smooth transitions, I always forget to even think about them 👏